### PR TITLE
RC1 update

### PR DESCRIPTION
**Pull request by jbomhold3**
Verified building the last pr was a package ref issue.

### CODE REVIEW

File: src/Blazor.Auth0.Shared/Blazor.Auth0.Shared.csproj [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```
Comment (henalbrod) [inline src/Blazor.Auth0.Shared/Blazor.Auth0.Shared.csproj]: The official recommendation is to target the lowest version of .NET Standard possible, and there's no known reason for upgrading it, please revert this change.

File: src/Blazor.Auth0.ClientSide/Blazor.Auth0.ClientSide.csproj [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```
Comment (henalbrod) [inline src/Blazor.Auth0.ClientSide/Blazor.Auth0.ClientSide.csproj]: The official recommendation is to target the lowest version of .NET Standard possible, and there's no known reason for upgrading it, please revert this change.

File: src/Blazor.Auth0.ClientSide/Authentication.cs [hunk 3]
```diff
@@ -94,7 +94,7 @@ public static string BuildAuthorizeUrl(AuthorizeOptions buildAuthorizedUrlOption
         /// <param name="uriHelper">A <see cref="IUriHelper"/> param.</param>
         /// <param name="authorizeOptions">A <see cref="AuthorizeOptions"/> param.</param>
         /// <returns>A <see cref="Task"/> representing the result of the asynchronous operation.</returns>
-        public static async Task Authorize(IJSRuntime jsRuntime, IUriHelper uriHelper, AuthorizeOptions authorizeOptions)
+        public static async Task Authorize(IJSRuntime jsRuntime, NavigationManager uriHelper, AuthorizeOptions authorizeOptions)
```
Comment (henalbrod) [inline src/Blazor.Auth0.ClientSide/Authentication.cs]: Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?

File: src/Blazor.Auth0.ClientSide/AuthenticationService.cs [hunk 4]
```diff
@@ -25,13 +26,12 @@ namespace Blazor.Auth0
     public class AuthenticationService : IAuthenticationService, IDisposable
     {
         private readonly ClientOptions clientOptions;
-        private readonly IUriHelper uriHelper;
+        private readonly NavigationManager uriHelper;
```
Comment (henalbrod) [inline src/Blazor.Auth0.ClientSide/AuthenticationService.cs]: Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?

File: src/Blazor.Auth0.ClientSide/AuthenticationService.cs [hunk 5]
```diff
@@ -77,16 +77,15 @@ private set
         /// <param name="uriHelper">A <see cref="IUriHelper"/> param.</param>
         /// <param name="options">A <see cref="ClientOptions"/> param.</param>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.OrderingRules", "SA1201:Elements should appear in the correct order", Justification = "I like this best ;)")]
-        public AuthenticationService(ILogger<AuthenticationService> logger, IComponentContext componentContext, HttpClient httpClient, IJSRuntime jsRuntime, IUriHelper uriHelper, ClientOptions options)
+        public AuthenticationService(ILogger<AuthenticationService> logger,  HttpClient httpClient, IJSRuntime jsRuntime, NavigationManager uriHelper, ClientOptions options)
```
Comment (henalbrod) [inline src/Blazor.Auth0.ClientSide/AuthenticationService.cs]: Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?

File: src/Blazor.Auth0.ServerSide/AuthenticationService.cs [hunk 6]
```diff
@@ -25,7 +26,7 @@ namespace Blazor.Auth0
     public class AuthenticationService : IAuthenticationService, IDisposable
     {
         private readonly ClientOptions clientOptions;
-        private readonly IUriHelper uriHelper;
+        private readonly NavigationManager uriHelper;
```
Comment (henalbrod) [inline src/Blazor.Auth0.ServerSide/AuthenticationService.cs]: Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?

File: src/Blazor.Auth0.ServerSide/AuthenticationService.cs [hunk 7]
```diff
@@ -70,7 +71,7 @@ private set
         public AuthenticationService(
             HttpClient httpClient,
             IHttpContextAccessor httpContextAccessor,
-            IUriHelper uriHelper,
+            NavigationManager uriHelper,
```
Comment (henalbrod) [inline src/Blazor.Auth0.ServerSide/AuthenticationService.cs]: Whenever possible, please make the property/field/parameter match its base-type name, would you rename this parameter, please?

**Comment by jbomhold3** [comment]
Changes made
